README.md

```diff
@@ -130,6 +130,31 @@ To dump VCDs (into the test_run_dir subfolder) using sbt:
 testOnly chiseltest.tests.BasicTest -- -DwriteVcd=1
 ```
 
+## Verilator Backend
+
+To run your tests using the [verilator](https://www.veripool.org/wiki/verilator) simulator:
+
+1. Ensure that you have a _supported version_ of verilator installed.
+   You can [consult the CI script](https://github.com/ucb-bar/chisel-testers2/blob/master/.github/workflows/test.yml#L38)
+   to find out which versions you can use.
+2. Enable the Verilator Backend by adding the right annotation to your tests:
```

Comment:
Perhaps clarify that this is on a per-`test()` basis (as opposed to project-wide or file-wide)?

Reply:
like "enable the Verilator backend for your individual tests by adding the right annotation:" ?

Reply:
Yeah, that's more clear